pypyr/dsl.py

```diff
@@ -645,8 +647,11 @@ def run_step(self, context):
         if self.description:
             description = context.get_formatted_value(self.description)
             run_me = context.get_formatted_as_type(self.run_me, out_type=bool)
-            skip_me = context.get_formatted_as_type(self.skip_me,
-                                                    out_type=bool)
+
+            skip_me = False
```

Comment:
?

Reply:
I have set it to false as the default value.

`skip_me` value is updated two lines below with an extra check of the `run_me` value, to prevent skip_me evaluation if `run_me` is False

Reply:
ah, gotcha! clever!